OpenRA.Mods.Common/UpdateRules/Rules/RenameSelfHealing.cs

```diff
@@ -0,0 +1,44 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RenameSelfHealing : UpdateRule
+	{
+		public override string Name { get { return "SelfHealing was renamed as negative SelfHealing is a common usecase."; } }
+		public override string Description
+		{
+			get
+			{
+				return "SelfHealing was renamed to ChangesHealth\n" +
+						"HealIfBelow was renamed to StartIfBelow.";
```

Comment:
Minor style nit: One level of indentation too much.

Reply:
Looks fine in Visual Studio Code with EditorConfig plugin:
![image](https://user-images.githubusercontent.com/756669/80387575-174d1680-88a9-11ea-88df-a9d50f97a122.png)

Reply:
You used two tabs for indentation. We usually only use one.

Reply:
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/UpdateRules/Rules/MoveClassicFacingFudge.cs#L25 I copied an existing rule and kept most of the formatting.

Reply:
Meh, I see. I guess it doesn't really matter that much anyway since this is only an update rule.